internal/pay/data.go

```diff
@@ -63,23 +67,33 @@ type ThreeDSecure struct {
 }
 
 type RefundSummary struct {
-	Status          string `json:"status"`
-	AmountAvailable int    `json:"amount_available"`
+	Status          string      `json:"status"`
+	AmountAvailable AmountPence `json:"amount_available"`
 }
 
 type SettlementSummary struct {
-	CaptureSubmitTime string `json:"capture_submit_time"`
-	CapturedDate      string `json:"captured_date"`
-	SettledDate       string `json:"settled_date"`
+	CaptureSubmitTime time.Time `json:"capture_submit_time"`
+	CapturedDate      date.Date `json:"captured_date"`
+	SettledDate       date.Date `json:"settled_date"`
+}
+
+type AmountPence int
```

Comment:
This is a very good idea, we have been lacking a nice money type. I think renaming `Pounds()` to `String()` and `Int()` to `Pence()` would be nicer. Also if you changed `DonorProvidedDetails.FeeAmount` to return this type we'd be able to remove the `penceToPounds` template fn.

Reply:
Nice, will update.